### PR TITLE
docker container: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added `docker container` missing command.

### CODE REVIEW

**Comment by Aracki** [comment]
@pxgamer You are right. Fixed.

File: pages/common/docker-containers.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# docker container
+
+> Manage Docker containers.
+> More information: <https://docs.docker.com/engine/reference/commandline/container/>.
+
+- List currently running Docker containers:
+
+`docker container ls`
+
+- Start one or more stopped containers:
```
Comment (sbrl) [inline pages/common/docker-containers.md]: How do you specify which container to start?

Comment (Aracki) [reply pages/common/docker-containers.md]: @sbrl Thanks for helping me notice there is a missing argument.

File: pages/common/docker-containers.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker container start {{container_name}}`
```
Comment (agnivade) [inline pages/common/docker-containers.md]: Perhaps we should add another container here to show that multiple containers are supported. That way we have a mix of single and multiple containers to give the idea that both are allowed.

Comment (Aracki) [reply pages/common/docker-containers.md]: I have added that just in this one command (maybe it's enough).

**Pull request merged**